OmniBLE/PumpManager/PodState.swift

```diff
@@ -117,7 +117,9 @@ public struct PodState: RawRepresentable, Equatable, CustomDebugStringConvertibl
         }
         return false
     }
-    
+
+    var deliveryStatusVerified: Bool // this variable is not persistent across app restarts
```

Comment:
I strongly believe that we need this added state for safety verification to prevent possible 0x31 pod faults as it is an reliable end to end safety check that the pod isn't delivering any insulin that Loop might not be properly tracking for whatever reason.

Reply:
I agree we need to fix this issue. 100%, and I think this is most of the way there. What I'm asking for is a modification to not make two state variables with very similar intentions; to make the new variable just track the currently unhandled situation of detecting a status response that has conflicting information about temp basal/basal delivery state, and continue to use unacknowledgedCommand to track unacknowledged commands.

Reply:
In order to do certain common pod comms optimizations safely, Loop does need to have some new state to handle a status response indicating an active insulin delivery isn’t being properly tracked for *whatever* reason. Would reversing the sense of the non-persistent `deliveryStatusVerified` variable in this PR and renaming it to something like `deliveryStateInconsistencyDetected` be sufficient to address the concern that this new state appears too similar to what the unacknowledged (delivery) command code manages?